tests/integ/domain_test.py

```diff
@@ -46,20 +46,6 @@ def testBaseDomain(self):
         self.assertEqual(rsp.status_code, 200)
         self.assertEqual(rsp.headers["content-type"], "application/json; charset=utf-8")
 
-        # verify that passing domain as query string works as well
-        del headers["X-Hdf-domain"]
-        req += "?host=" + self.base_domain
-        rsp = self.session.get(req, headers=headers)
-        self.assertEqual(rsp.status_code, 200)
-        self.assertEqual(rsp.headers["content-type"], "application/json; charset=utf-8")
-
```

Comment:
Lines 49-54 should be able to stay, since this part is just testing that a regular (non-DNS) domain as a query string works.